### PR TITLE
Added meme-battles channel

### PR DESCRIPTION
**Pull request by DigTK**

### CODE REVIEW

File: _pages/channels/meme-battles.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+channel: meme-battles
+permalink: /channels/meme-battles/
+parent: channels
+---
+
+A recreational channel to share and discuss your favorite memes, or even create new ones. This is a no-judgement zone. No meme is too old, no pepe too sad. 
+
+Drop in to take your mind off of things and have some good laughs and afterwards you get back to your code fully revitalized.
+
+![there should be a sweet meme here but it is broken](/images/longcat-tacgnol.png "It's an older meme but it checks out")
```
Comment (agarzola) [inline _pages/channels/meme-battles.md]: Please use the alt text to describe the image. The title should have a period at the end.

File: _pages/channels/meme-battles.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: channel
+title: Meme-Battles
```
Comment (agarzola) [inline _pages/channels/meme-battles.md]: Let’s change the title to _Meme Battles_ (no hyphen).

File: _pages/channels/meme-battles.md (hunk 1 above)
Comment (StrangeWill) [inline _pages/channels/meme-battles.md]: That markdown sets the alt text for me.

Comment (agarzola) [inline _pages/channels/meme-battles.md]: Yes, it does, but the text does not describe the image, as it should.

Comment (StrangeWill) [inline _pages/channels/meme-battles.md]: But it _is_ and older meme and it _does_ check out.

Comment (agarzola) [inline _pages/channels/meme-battles.md]: That’s the `title`, not the `alt` text. But even that does not qualify as alt text. Being truthful `!==` being descriptive.

**Pull request merged**